styleguide/template.md

```diff
@@ -202,7 +202,35 @@ You can use a [Markdown table generator tool](http://www.tablesgenerator.com/mar
 
 ## Code
 
-### Code blocks with language identifier
+The best way to include code is to include snippets from a working sample. Create your
+sample following the instructions in the [contributing guide](../CONTRIBUTING.md#contributing-to-samples).
+
+You can include the code using include syntax:
+
+```
+[!code-csharp[<title>](<pathToFile>#<RegionName)]
+```
+
+The example above shows C# syntax, but other languages are supported.
+Use `code-fsharp` for F# samples; use `code-vbnet` for Visual Basic samples.
+Any language supported by [highlight.js](http://highlightjs.readthedocs.io/en/latest/css-classes-reference.html#language-names-and-aliases)
+is supported. 
+
+The text you place for `<title>` shows up as a rollover on the text. The `<pathToFile>`
+is the path to the source file. The `<RegionName>` should be a region in your source
+code that should be included. Use the `#region` and `#endregion` preprocessor syntax
+to specify the region of code to include.
+
+For cases where regions don't work, you can specify the start and end of a region
```

Comment:
"For cases where regions don't work, you can specify the start and end of a region" -> 
For cases where regions don't work, you can specify the start and end of your snippet? it's a bit confusing to say regions don't work, so specify a region